doc/api/worker_threads.md

```diff
@@ -46,7 +46,7 @@ if (isMainThread) {
 ```
 
 The above example spawns a Worker thread for each `parse()` call. In actual
-practice, use a pool of Workers instead for these kinds of tasks. Otherwise, the
+practice, use a pool of Workers instead of these kinds of tasks. Otherwise, the
```

Comment:
I think this is correct as its already written. However, if you're looking for something to change here, I'd drop "actual" from "actual practice."